packages/server/graphql/public/typeDefs/stopRecurrence.graphql

```diff
@@ -17,10 +17,9 @@ union StopRecurrencePayload = ErrorPayload | StopRecurrenceSuccess
 
 type StopRecurrenceSuccess {
   """
-  The updated meeting
-  :TODO: Change this to be any NewMeeting.
+  The updated meeting series
   """
-  meeting: TeamPromptMeeting!
+  meetingSeries: MeetingSeries!
```

Comment:
This is the biggest change in this PR. To handle all the cases, I think it makes much more sense to return `MeetingSeries` from `stopRecurrence` instead of just a `Meeting`. The main reason for this is that technically there's a possibility that none of the meetings are active, but we should still be able to disable the recurrence (last AC from the task). In this case, there's no meeting to return, as there are no active meetings. In general, `start/stop` recurrence mutations create/update `MeetingSeries` so I think it's a bit better to structure it that way.

Reply:
I'm not sure I agree with this, since it breaks the symmetry of starting recurrence (action on a meeting) and ending recurrence (now on a meeting series).

> In this case, there's no meeting to return, as there are no active meetings

I'd say the meeting to accept as an argument and then return would just be an ended meeting

Reply:
yeah, `startRecurrence` would have to be aligned to this. The biggest problem I have with passing `meetingId` in `stopRecurrence` input is the fact it can be used on the already ended meetings. what's more, there might be 0 active meetings, what should we return then, as a result? I think it'd be best to have some sync time and make sure we're on the same page with both PRs, shouldn't take too long. wdyt?

Reply:
I think ending on a meeting series is a safe bet. I can imagine that once we extend this to other meeting types we might want to have an overview for all meeting series and then be able to start/stop them.

Starting a meeting series should take a meeting as argument to take as a prototype to know which structure it has (i.e. which template to use etc.).

Restarting should take again a meeting series as that's the object we're mutating.